src/pybind/ceph_volume_client.py

```diff
@@ -239,6 +249,7 @@ def __init__(self, auth_id, conf_path, cluster_name, volume_prefix=None, pool_ns
         # UUID
         self._id = struct.unpack(">Q", uuid.uuid1().get_bytes()[0:8])[0]
 
+        self.version = _CEPHFSVOLUMECLIENT_VERSION
```

Comment:
We need two versions here, the latest version, and the earliest version we're compatible with.

We should also make this a class attribute, so that callers don't have to have the correct arguments to __init__ before they can query the version.